microsoftgraph/microsoft-graph-explorer-v4
# Fix: Stop clearing response when editing url

## Overview

Solves the reason why editing the sample url clears the query response.
Fixes #390 

### Notes

N/A

## Testing Instructions

* Run a sample
* Start changing the sample url
* Notice that the response does not get deleted
